src/pages/log-management/opensearch/opensearch-dashboards/api.mdx

```diff
@@ -0,0 +1,86 @@
+</Callout>
+
+Each request to a REST API Endpoint must include all of the necessary information for Opensearch Dashboards to fulfill the request. If all the necessary 
+information is provided the API requests return JSON output.
+
+Use the following operations to interact with the Opensearch Dashboards via the Logit.io Proxy.
+
+* __GET__ - Fetches the information.
+* __POST__ - Adds new information.
+* __PUT__ - Updates the existing information.
+* __PATCH__ - Applies partial modifications to the existing information.
+* __DELETE__ - Removes the information.
+
+All the details that you require in order to connect to the Logit.io proxy are found on the Profile Settings page of the Logit.io Dashboard. 
+The Profile Settings page has a section called `Proxy Settings` where you will find everything that you need. 
+Click [here](https://dashboard.logit.io/profile) to view you profile page.
+
+
+There are two ways that you can call the REST API using the Logit.io proxy. The call to our proxy needs to be in either of the following formats:
+
+__Basic Authentication header__:
+
+```bash
+curl -XGET "osd-xsrf:true" -H "Content-Type: application/json" 
+-H "Authorization: Basic {basic auth header}" {endpoint to call}
+```
+
+__Proxy Username and Password__:
+
+```bash
+curl -XGET "osd-xsrf:true" -H "Content-Type: application/json" 
+-u "{proxy username}:{proxy password}" {endpoint to call}
+```
+
+Below are two curl request examples that show how to connect to the Logit.io Proxy. The examples use the Stack Id of a Stack that you have access to. 
+If you need to access another of your stacks then you can simply find the Stack that you require, retrieve the stack Id from that stack and then 
+replace it. 
+
+<ProxyAuthPlaceholderWarning stackType="logs" />
+
+### Example of a curl request to access your Logit.io Opensearch Dashboards instance
+
+This first method uses the __Basic Auth Header__:
+
+```bash copy /@proxyAuthSetting.basicAuthHeader/ /@logs_id/
+curl -XGET -H "osd-xsrf:true" -H "Content-Type: application/json" -H ^
+"Authorization: Basic ^
+@proxyAuthSetting.basicAuthHeader" ^
+https://kibana.logit.io/s/@logs_id/api/status
+```
+
+This second method uses the __proxy username and password__:
+
+```bash copy /@proxyAuthSetting.username/ /@proxyAuthSetting.password/ /@logs_id/
+curl -XGET -H "osd-xsrf:true" -H "Content-Type: application/json" ^
+-u "@proxyAuthSetting.username:@proxyAuthSetting.password" ^
+https://kibana.logit.io/s/@logs_id/api/status
+```
+
+Both curl requests call the same API Endpoint. This request returns detailed information about your Opensearch Dashboards instance’s health including the 
+resource usage, the server status, installed plugins, plugin statuses and metrics.
+
+There are many other Opensearch Dashboards features that can be accessed via REST API through the Logit.io Proxy. These features are explained in the 
+Elastic docs [here](https://opensearch.org/docs/latest/api-reference/)
```

Comment:
I wonder whether we should include an example within our site rather than just off-loading the user to Opensearch / Elasticsearch?

Reply:
I was going to go that way at first but after speaking with Lee it was decided to link to the docs. There is an example on the page, the link is to get other endpoints that they can call.